.circleci/config.yml

```diff
@@ -31,15 +31,37 @@ jobs:
       - run: composer require silverstripe/recipe-cms
       - run: composer update -n --prefer-dist --ignore-platform-reqs
       - run: composer vendor-expose
+      - run:
+          name: Setup Integration Pre-requisites
+          command: |
+            if [ ! -d assets ]; then mkdir assets; fi
+            cp .circleci/.env.test ./.env
+            cp tests/mocks/TestIndex.php app/src/CircleCITestIndex.php
+            sed -i -e 's/namespace Firesphere\\SolrSearch\\Tests;//' app/src/CircleCITestIndex.php
+            sed -i -e "s/ implements TestOnly//" app/src/CircleCITestIndex.php
+            sed -i -e "s/TestIndex/CircleCITestIndex/" app/src/CircleCITestIndex.php
       - run: vendor/bin/sake dev/build flush=all skipannotation=true
       - run:
           name: Run Solr Configure
           command: |
             service apache2 restart
             service solr restart
             mkdir -p /var/www/html/.solr && chmod -R 777 /var/www/html/.solr
-            vendor/bin/sake dev/tasks/SolrConfigureTask
-      - run: vendor/bin/sake dev/tasks/SolrIndexTask
+            chown -R solr:solr /var/www/html/.solr
```

Comment:
Could you extract this block in to a bash script in `tests/solr` and then run the bash script? It would keep the config a bit cleaner :)

Reply:
Thanks 👍 Done